src/Http/Controller/Admin/AdvsController.php

```diff
@@ -205,4 +207,21 @@ public function advancedUpdate()
             }
         }
     }
+
+
+    public function queryAdvs(AdvRepository $advRepository, AdvModel $advModel)
```

Comment:
queryAdvs is not a clean method name I think. You can find something like searchAdvsByName or anything compatible.